### PR TITLE
refine openvino loadtf exception handling and log output

### PR DESCRIPTION
**Pull request by glorysdj**
this pr is to refine openvino loadtf exception handling and log output

---
This change is [Reviewable](https://reviewable.io/reviews/intel-analytics/analytics-zoo/1183)

### CODE REVIEW

File: zoo/src/main/scala/com/intel/analytics/zoo/pipeline/inference/OpenVinoInferenceSupportive.scala [hunk 1]
```diff
@@ -108,26 +108,42 @@ object OpenVinoInferenceSupportive extends InferenceSupportive {
...
           outputPath,
-          motfpyFilePath) #> log !
+          motfpyFilePath) ! log
+      }
+      logger.info(s"tensorflow model optimized, log: \n" +
+        s"stderr: $stderr \n" +
+        s"stdout: $stdout \n" +
+        s"exitCode: $exitCode\n -----")
+      exitCode match {
+        case 0 => logger.info(s"tensorflow model optimize successed")
```
Comment (hkvision) [inline zoo/src/main/scala/com/intel/analytics/zoo/pipeline/inference/OpenVinoInferenceSupportive.scala]: typo... succeeded

**Comment by hkvision** [comment]
LGTM

**Comment by glorysdj** [comment]
passed in http://10.239.47.211:18888/job/ZOO-PR-Validation/1925/

File: zoo/src/main/scala/com/intel/analytics/zoo/pipeline/inference/OpenVinoInferenceSupportive.scala (hunk 1 above)
Comment (glorysdj) [reply zoo/src/main/scala/com/intel/analytics/zoo/pipeline/inference/OpenVinoInferenceSupportive.scala]: ok

**Pull request merged**